templates/Compensate_Image_Data.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    for multiplier in multipliers:
</pre><p>this looping could probably be moved into <code>rosetta.compensate_image_data()</code>which can take <code>multipliers</code>  as an argument instead of an <code>out_dir</code>  and a <code>mat_path</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Right. The issue is that after using this notebook to pick coefficients for the rosetta matrix, the <code>compensate_image_data</code>  function will then be run across all of the cohort, but with only a single rosetta matrix. That one probably won't be named according to this framework</p>